src/Forms/UrlField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+/**
+ * Text input field with validation for a url using filter_var($url, FILTER_VALIDATE_URL)
+ * The value of the field must start with a protocol such as http:// or https://
+ */
+class UrlField extends TextField
+{
+    protected $inputType = 'text';
+
```

Comment:
```suggestion
```
This is already provided as the default value in `FormField` - even `TextField` doesn't redeclare it. No need to redeclare it.

Reply:
Removed